Packs/Devo/Integrations/Devo_v2/Devo_v2.yml

```diff
@@ -4,193 +4,212 @@ commonfields:
-      description: End datetime for specified query. If provided must be in same format as "from" argument. This argument is ignored in a date range.
-    - name: limit
-      description: Limit of results to return to context. 0 for no limit.
-      defaultValue: "50"
-    - name: queryTimeout
-      description: Timeout in seconds for this query to run against Devo to override the minute default in the platform.
-      defaultValue: "60"
-    - name: writeToContext
-      description: Whether to write results to context. Can be "true" or "false". The default value is "true".
-      defaultValue: "true"
-    outputs:
-    - contextPath: Devo.MultiResults
-      description: A list of dictionary results.
-    description: Queries multiple tables for a given token and returns relevant results.
-  - name: devo-write-to-table
-    arguments:
-    - name: tableName
-      required: true
-      description: The name of the table to write to.
-    - name: records
-      required: true
-      description: Records to write to the specified table.
-      isArray: true
-    - name: linqLinkBase
-      description: Overrides the global Devo base domain for linq linking.
-    outputs:
-    - contextPath: Devo.RecordsWritten
-      description: Records written to specified Devo table.
-      type: unknown
-    - contextPath: Devo.LinqQuery
-      description: The LINQ query to use to see your data in Devo.
-    - contextPath: Devo.QueryLink
-      description: The link to the Devo table for the executed query.
-    description: Writes records to a specified Devo table.
-    execution: true
-  - name: devo-write-to-lookup-table
-    arguments:
-    - name: lookupTableName
-      required: true
-      description: The lookup table name to write to.
-    - name: headers
-      required: true
-      description: Headers for lookup table control.
-      isArray: true
-    - name: records
-      required: true
-      description: Records to write to the specified table.
-      isArray: true
-    outputs:
-    - contextPath: Devo.RecordsWritten
-      description: Lookup records written to the lookup table.
-      type: unknown
-    description: Writes lookup table entry records to a specified Devo table.
-    execution: true
-  dockerimage: demisto/devo:1.0.0.47005
+    - name: devo-run-query
+      arguments:
+        - name: query
+          required: true
+          description: A LINQ query to run in Devo, with pagination support.
+        - name: from
+          required: true
+          description: Start datetime for the specified query. This argument supports natural language (e.g., 2 day, 3 week), Unix timestamps, Python datetime objects, and string datetimes.
+        - name: to
+          description: End datetime for specified query. If provided must be in same format as "from" argument. This argument is ignored in a date range.
+        - name: items_per_page
+          description: Enter the per page value you want to set.
+          required: true
```

Comment:
Can't be required as it's not backwards compatible. change it to "false" and give it a default value.